assets/prefabs/recipe/RecipePeachJuice.prefab

```diff
@@ -0,0 +1,29 @@
+{
+    "ProcessDefinition": {
+        "processType": "Cooking:Cooking"
+    },
+    "CraftingStationRecipe": {
+        "recipeId": "Cooking|Cooking:PeachJuice",
+
+        "recipeComponents": ["2*Cooking:peach","1*Cooking:sugar"],
+        "recipeFluids": ["0.6*Fluid:Water"],
+        "requiredTemperature": 0,
+        "processingDuration": 4500,
+
+        "itemResult": "1*Cooking:PeachJuice"
+    },
+    "ProcessingTime": {
+        "duration": 9000
```

Comment:
Oh yeah sure thing, I was actually changing this for a task I intended to do immediately after the skewer one, so I'll claim the task, fix this, and submit.